### ISSUE TITLE
Add support for Quartz format (6 digits)

### ISSUE THREAD

**Issue by karoletrych** [karoletrych/vue-cron-editor#1]
<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Oni07R** [comment]
Any news about this issue?

**Comment by Oni07R** [comment]
These are the fixes that I have to manually apply to comply to quartz format:
- add one * at the start the expression for seconds
- switch any */ with 0/
- * for both DayOfWeek and DayOfMonth is not supported, one has to be ?

**Comment by karoletrych** [comment]
I'll try to add a couple of options that will allow producing Quartz compatible expressions:
- add seconds on the first position to generated expressions (0 by default)
- make seconds editable
- replace DayOfWeek or DayOfMonth (depending on the tab selected with '?')
- add aliases for days of week (SUN, MON instead of 1,2)

Users will be able to specify the 'flavor' of cron (set of default options, 'basic' or 'quartz' for now) and override some of the options if necessary f.e.
```
<VueCronEditor
 :flavor="quartz"
 :isSecondsVisible="false"
/>
```


> switch any */ with 0/
Are you sure that it's required by Quartz?
I've typed '0 0 */1 * * ?' here: https://www.freeformatter.com/cron-expression-generator-quartz.html and it says its valid.

**Comment by Oni07R** [comment]
> Are you sure that it's required by Quartz?
> I've typed '0 0 */1 * * ?' here: https://www.freeformatter.com/cron-expression-generator-quartz.html and it says its valid.

I am new to Quartz but all examples here
http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html
use 0/ and our back-end returns error when sending */

**Comment by tsteenkamp** [comment]
Any progress on this? :)

**Comment by tsteenkamp** [comment]
Thank you very much for the effort so far! For it me it does not work with an initial value though, or am I missing something?
For `:cronSyntax="'basic'"` an initial value of v-model from the example `4 4 * * 0,2,3,5` works. For `:cronSyntax="'quartz'"` and and initial value of `'0 0/1 * * * ?` does not work and v-model gets set to `null`. Maybe some race condition and/or the syntax is set to quartz after the initial expression is validated or something similar?

**Comment by karoletrych** [comment]
Yup, found a nasty bug in expression parser. Fixed in 0.2.17.

**Comment by tsteenkamp** [comment]
awesome, seems to work :) thx a lot!
